samwise.js

```diff
@@ -3,41 +3,40 @@
  *
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.